conf/route_control.py

```diff
@@ -3,528 +3,884 @@
+                    "Module %s:%i->%i/%s connected",
+                    module,
+                    ogate,
+                    igate,
+                    next_module,
+                )
+                break
+            finally:
+                self.bess.resume_all()
         else:
-          print("The IP address {} is valid ipv6 address. Ignore ".format(ipb))
-          return
-    except:
-        print("The IP address {} is invalid".format(item.neighbor_ip))
-        return
-    # Probe ARP request by sending ping
-    send_ping(item.neighbor_ip)
-
-    # Probe ARP request
-    ##send_arp(neighbor_ip, src_mac, item.iface)
-
-
-def parse_new_route(msg):
-    item = NeighborEntry()
-    # Fetch prefix_len
-    item.prefix_len = msg['dst_len']
-    # Default route
-    if item.prefix_len == 0:
-        item.iprange = '0.0.0.0'
-
-    for att in msg['attrs']:
-        if 'RTA_DST' in att:
-            # Fetch IP range
-            # ('RTA_DST', iprange)
-            item.iprange = att[1]
-        if 'RTA_GATEWAY' in att:
-            # Fetch gateway MAC address
-            # ('RTA_GATEWAY', neighbor_ip)
-            item.neighbor_ip = att[1]
-            _mac = fetch_mac(att[1])
-            if not _mac:
-                gateway_mac = 0
+            raise Exception(
+                "BESS module connection ({}:{}->{}:{}) failure.".format(
+                    module, ogate, igate, next_module
+                )
+            )
+
+    def del_module(self, module_name) -> None:
+        """Deletes a BESS module.
+
+        Args:
+            update_module (str): The name of the module to delete.
+        """
+        for _ in range(MAX_RETRIES):
+            try:
+                self.bess.pause_all()
+                self.bess.destroy_module(module_name)
+            except Exception:
+                logger.exception(
+                    "Error destroying module %s. Retrying in %i sec...",
+                    module_name,
+                    SLEEP_S,
+                )
+                time.sleep(SLEEP_S)
             else:
-                gateway_mac = mac2hex(_mac)
-        if 'RTA_OIF' in att:
-            # Fetch interface name
-            # ('RTA_OIF', iface)
-            item.iface = ipdb.interfaces[int(att[1])].ifname
-
-    if not item.iface in args.i or not item.iprange or not item.neighbor_ip:
-        # Neighbor info is invalid
-        del item
-        return
-
-    # if mac is 0, send ARP request
-    if gateway_mac == 0:
-        print('Adding entry {} in arp probe table. Neighbor: {}'.format(item.iface,item.neighbor_ip))
-        probe_addr(item, ipdb.interfaces[item.iface].address)
-
-    else:  # if gateway_mac is set
-        print('Linking module {}Routes with {}Merge (Dest MAC: {})'.format(
-            item.iface, item.iface, _mac))
-
-        link_route_module(bess, gateway_mac, item)
+                logger.info("Module %s destroyed", module_name)
+                logger.info("BESS resume all")
```

Comment:
This log doesn't match with what has been done

Reply:
moved